nautobot/core/templates/inc/custom_fields/panel_data.html

```diff
@@ -1,39 +1,60 @@
 {% load helpers %}
 {% if custom_fields %}
+<style>
+    .accordion-toggle {
+        font-size: 15px;
```

Comment:
Looks like the rest of the elements in this table are all defaulting to 14px, including the panel header, so let's make that consistent.